apispec/ext/marshmallow/swagger.py

```diff
@@ -41,6 +41,32 @@
     marshmallow.fields.List: ('array', None),
 }
 
+CUSTOM_FIELD_MAPPING_ATTR = '__custom_swagger_field_mapping'
```

Comment:
primarily b/c one is not supposed to actually read/use the attribute by hand, so there's no percentage in making the name shorter. `__swagger_field_mapping` would definitely be better for an attribute that was part of the api. I'll change if you like tho.